classes/helper/HelperForm.php

```diff
@@ -26,8 +26,8 @@
  use PrestaShop\PrestaShop\Core\Addon\Module\ModuleManagerBuilder;
 
  /**
-  * @since 1.5.0
-  */
+ * @since 1.5.0
```

Comment:
this alignment doesn't seem good.. did it come from php-cs-fixer?

Reply:
Yes, it did. I agree with you! But it solved the lint issue reported by travis.

Reply:
I fixed it.